style-guide.md

```diff
@@ -122,4 +122,4 @@ This document is meant to clarify some of the implicit conventions adopted in WC
 - Ranking/Rank: An ordinal number representing a result's order in a list of results (e.g. ranking in a round, world rank).
 - Record: A best-ranked result (e.g. of comparable results for a region, or of results for a competitor)
 - Part: A subset of puzzle components (in particular, any individual piece, cap, sticker, etc.).
-- Piece: A part of the puzzle that stays together during normal use of a puzzle (e.g. a cubie with its stickers).
+- Piece: A part of the puzzle that stays together during normal use of a puzzle (e.g. a cube with its stickers).
```

Comment:
This is specifically using the cuber name of "cubie" for a cubic piece to give an example. It should stay.
```suggestion
- Piece: A part of the puzzle that stays together during normal use of a puzzle (e.g. a cubie with its stickers).
```